account_manager/src/validator/slashing_protection.rs

```diff
@@ -168,18 +167,17 @@ pub fn cli_run<T: EthSpec>(
                     }
                 }
                 Err(InterchangeError::AtomicBatchAborted(outcomes)) => {
-                    eprintln!("ERROR, slashable data in input:");
+                    eprintln!("ERROR: import aborted due to one or more errors");
                     for outcome in &outcomes {
                         if let InterchangeImportOutcome::Failure { pubkey, error } = outcome {
                             eprintln!("- {:?}", pubkey);
                             eprintln!("    - error: {:?}", error);
                         }
                     }
-                    return Err(
-                        "ERROR: import aborted due to slashable data, see above.\n\
-                         Please see https://lighthouse-book.sigmaprime.io/slashing-protection.html#slashable-data-in-import\n\
-                         IT IS NOT SAFE TO START VALIDATING".to_string()
-                    );
+                    return Err("ERROR: import aborted due to errors, see above.\n\
+                                No data has been imported and the slashing protection \
+                                database *should* be in the same state it was in before the import.\n\
```

Comment:
This branch is now only reachable if the database itself refuses the write, which it shouldn't do unless there's an external issue. In such a case I wanted to avoid making any guarantees about the database's state, because it's possible for it to end up corrupted through no fault of Lighthouse, e.g. memory corruption, filesystem corruption, the user deleting the file mid operation, etc. Maybe that's being too cautious though. The only way I was able to reach this branch in testing was by setting the database read-only, in which it does indeed remain the same after the failure

Reply:
Maybe something like:

" ... database is in the same state as before the import started. Except for unlikely external issues, such as memory corruption or user manipulation. So, if the `valatator_definitions.yml` and keystones are reverted to the same state as when the validator was last running, the validator can be started again."

If I'd had read that, I probably would not have used `--init-slashing-protection` when I saw this message.

Reply:
As discussed offline, I've tweaked this message to remove the "should" but avoided making any positive statements about starting the VC, as this error should be so rare that it's an "all bets are off" sort of scenario.